modules/standard/Types.chpl

```diff
@@ -346,52 +469,95 @@ proc chpl__commonType(type s, type t) type
 //
 // numBits(type) -- returns the number of bits in a type
 //
-
+// joint documentation, for user convenience
+/*
+Returns the number of bits used to store the values of type `t`.
+This is available for all numeric types, fixed-width `bool` types,
+and ``enum`` types.
+It is not available for default-width `bool`.
+*/
 proc numBits(type t) param where t == bool {
   compilerError("default-width 'bool' does not have a well-defined size");
 }
+pragma "no doc"
 proc numBits(type t) param where t == bool(8) return 8;
+pragma "no doc"
 proc numBits(type t) param where t == bool(16) return 16;
+pragma "no doc"
 proc numBits(type t) param where t == bool(32) return 32;
+pragma "no doc"
 proc numBits(type t) param where t == bool(64) return 64;
+pragma "no doc"
 proc numBits(type t) param where t == int(8) return 8;
+pragma "no doc"
 proc numBits(type t) param where t == int(16) return 16;
+pragma "no doc"
 proc numBits(type t) param where t == int(32) return 32;
+pragma "no doc"
 proc numBits(type t) param where t == int(64) return 64;
+pragma "no doc"
 proc numBits(type t) param where t == uint(8) return 8;
+pragma "no doc"
 proc numBits(type t) param where t == uint(16) return 16;
+pragma "no doc"
 proc numBits(type t) param where t == uint(32) return 32;
+pragma "no doc"
 proc numBits(type t) param where t == uint(64) return 64;
+pragma "no doc"
 proc numBits(type t) param where t == real(32) return 32;
+pragma "no doc"
 proc numBits(type t) param where t == real(64) return 64;
+pragma "no doc"
 proc numBits(type t) param where t == imag(32) return 32;
+pragma "no doc"
 proc numBits(type t) param where t == imag(64) return 64;
+pragma "no doc"
 proc numBits(type t) param where t == complex(64) return 64;
+pragma "no doc"
 proc numBits(type t) param where t == complex(128) return 128;
 
 //
 // numBytes(type) -- returns the number of bytes in a type
 //
 
+pragma "no doc" // do we want to document this one?
```

Comment:
Yes, we do.  We'd like to encourage the user to use a system-defined bit-per-byte constant (even though it is never expected to change) rather than the manifest constant 8 all over the place.